CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# 3.1.0
```

Comment:
I would mark this release as a `major` actually because we cant guarantee that it wont break if users are using React v16. But I would specifiy that i might not be breaking.

Reply:
I use the component with react 16.1.1 and it works as expected 🎆

Reply:
Ah...did not see the use of React v16 in dev-deps. Sry, my bad

Reply:
Yep! :)